### ISSUE TITLE
MacOS Safari HLS/MSE black screen, only WebRTC works

### ISSUE THREAD

**Issue by HausnerR** [AlexxIT/WebRTC#563]
Hello,
Strange issue - HLS and MSE does not work (connected but black screen), but WebRTC works like a charm.

Checking logs I see that in MSE script connects to stream and pulls data but looks like not displays it.

<img width="1680" alt="Zrzut ekranu 2023-09-3 o 12 33 56" src="https://github.com/AlexxIT/WebRTC/assets/5604289/775649f5-84c4-4c8d-af43-289d64b8ea46">

**Comment by HausnerR** [comment]
Found out that on rtsp integration works, black screen on MSE and HLS occurs only when tapo integration is used and only in Safari (Chrome works normally, Firefox not checked)

**Comment by flyingpeter** [comment]
Hey i got same issue. I think its related. 
Here is my problem:
Windows/chrome or firefox: no issues
Android no issues
Iphone: works away from my lan in ios app.
Doesnt work on lan (even with vpn)

When its outside my lan its says RTC and its working. 
When on lan keeps trying to load and nothing happens.

Any idea? Thanks!

**Comment by AlexxIT** [comment]
Yes. MSE may have problems for problem stream. WebRTC works better for bad stream. Maybe will be fixed someday
